docs/_includes/css/forms.html

```diff
@@ -261,32 +261,32 @@ <h2 id="forms-controls">Supported controls</h2>
   <div class="bs-example" data-example-id="block-checkboxes-radios">
     <form>
       <div class="checkbox">
-        <label>
-          <input type="checkbox" value="">
+        <label for="optionsCheckBox1">
+          <input type="checkbox" id="optionsCheckBox1" value="">
           Option one is this and that&mdash;be sure to include why it's great
         </label>
       </div>
```

Comment:
Having a ``<label>`` wrap around a form control and its text is a valid construct, and works consistently in all assistive technologies.